privacyidea/lib/tokens/pushtoken.py

```diff
@@ -474,6 +490,10 @@ def create_challenge(self, transactionid=None, options=None):
                                                  ACTION.CHALLENGETEXT,
                                                  options) or DEFAULT_CHALLENGE_TEXT
 
+        sslverify = get_action_values_from_options(SCOPE.AUTH,
+                                                   PUSH_ACTION.SSL_VERIFY,
+                                                   options) or "1"
+
```

Comment:
Again, I think it would be nice to check that ``sslverify`` is really either ``"1"`` or ``"0"``.

Reply:
OK

Reply:
done in f1ae0319

Reply:
Heh, using ``getParam`` like that is clever, but in my opinion it's not very readable :-/ Also, I'm not sure if it makes sense to add the ``allowed_values`` feature to ``getParam`` just for this occasion.
What do you think about just adding a check
```python
if sslverify not in ("0", "1"):
    raise ParameterError("Invalid value for sslverify: {!r}".format(sslverify))
```
for both occasions of ``sslverify``? I realize it's a bit boilerplate, but I think it's more readable. What do you think?

Reply:
It is more readable. And actually I already had added small function in lib/utils.py to do this - including tests. But then I though it makes more sense in the parameters, especially since this might not be the last API parameter, which we should check for valid values!